dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPI.java

```diff
@@ -38,4 +39,8 @@ Experiment archive(final String id, final User user)
      */
     void delete(String id, User user) throws DotDataException, DotSecurityException;
 
+    /**
+     * Returns experiments based on the provided filters in {@link ExperimentFilter}
+     */
+    List<Experiment> list(final ExperimentFilter filter, final User user) throws DotDataException;
```

Comment:
I think we can improve the doc here, for example which are the filter that we can apply

Reply:
Done